### DISCUSSION TITLE
最新版amll插件端歌词显示bug

### DISCUSSION THREAD

**Discussion by HaoZi9167** [amll-dev/applemusic-like-lyrics#258] [Q&A]
https://github.com/user-attachments/assets/d3452045-3c89-4e06-8009-9d29eb375dd6

如上，选择TTML歌词库会导致歌词不可见

**Comment by SheldonHomes** [comment]
同问题

**Reply by OnismIS** [reply]
一样，调试里面歌词数据为空

**Comment by Steve-xmh** [comment]
插件端已经终止开发，还请谅解

**Comment by yzbujin** [comment]
https://amlldb.bikonoo.com/mirror.html
添加一下就行了
